### ISSUE TITLE
Two-Pane Layout Improvements

### ISSUE THREAD

**Issue by jjspace** [a85/POSTMan-Chrome-Extension#2716]
I just discovered the two-pane layout added in 4.9 and I really like it. It really increases the screen usage and reduces scrolling and allows me to see more of very long responses. However I think it still needs a little adjustment. The biggest thing I noticed was that the url bar got shrunk way down so it was hard to read the entirety of long request urls (as shown in picture 1).

I would propose to return the width of the url bar and moving the status back down to the end of the response tab bar (as shown in picture 2).

![Problems](http://i.imgur.com/wqHSEPE.png)
![Solution](http://i.imgur.com/PXxTEGh.png)

**Comment by sdnts** [comment]
@jjspace Thanks for the suggestion :) We'll look into this

**Comment by mastermind3043-zz** [comment]
@madebysid @vkaegis alternately, can the right pane be made collapsible in the two-pane view. 
Such that, when composing the request, one can collapse the response pane

**Comment by m-stilling** [comment]
Any update on this? :) It's currently an absolute pain to edit the URL when you snap the window to take up half a screen.

**Comment by Kanaduchi** [comment]
What is update of this issue? Issue was opened a year ago. 

Currently it is very uncomfortable to work in two-side view with long links

**Comment by m-stilling** [comment]
@madebysid @vkaegis Can we have an update on this? :)

**Comment by vkaegis** [comment]
Hey guys! Apologies for the long delay on an update. We've been waiting on some restructuring to be able to accommodate this. We're finally taking a closer look at the entire request tab UI for the upcoming release and hope to allow collapsible panes (as suggested by @mastermind3043) soon after. Will keep you updated!

**Comment by m-stilling** [comment]
@vkaegis Can't you close this now? Also thanks a ton! :)

**Comment by travi** [comment]
@mich356c thanks for the comment. got me to download the latest version and see the update :)

so much better. thank you everyone for getting that taken care of 🎉

**Comment by vkaegis** [comment]
Awesome. Feel free to open another issue if you have more feedback around this. Closing this for now.

**Comment by curtisgibby** [comment]
In the future, it might be helpful to include a comment on an issue like this, saying something along the lines of "Fixed in version 6.5.2, released on November 1, 2018" or whatever, so that people who've been following this issue know that they can now get a new version that resolves their concern.

**Comment by vkaegis** [comment]
@curtisgibby Thanks for the feedback. This particular fix was released with v6.4.0 of the native apps on the 8th of October, 2018.
